core/src/main/java/com/radixdlt/addressing/Addressing.java

```diff
@@ -133,14 +133,14 @@ public ComponentAddress decodeAccountAddress(String address) {
         Bech32mCoder.decodeWithExpectedHrp(network.getAccountComponentHrp(), address));
   }
 
-  public String encodeSystemAddress(SystemAddress systemAddress) {
+  public String encodeSystemAddress(ComponentAddress systemAddress) {
```

Comment:
For the current version of scrypto lib, while the SystemAddress has been removed, SystemHrp has not been properly removed. Validator, EpochManager and Clock remain system HRP. I think this will be addressed in a future PR.